elixir/apps/domain/test/domain/resources_test.exs

```diff
@@ -367,33 +365,6 @@ defmodule Domain.ResourcesTest do
       refute Map.has_key?(errors_on(changeset), :address)
     end
 
-    test "does not allow cidr addresses to overlap for the same account", %{
```

Comment:
@AndrewDryga, also removed this test as it didn't seem necessary based on one of your commits in this PR.